src/sentry/rules/actions/notify_event_sentry_app.py

```diff
@@ -102,7 +102,14 @@ def self_validate(self) -> None:
         schema = alert_rule_component.schema.get("settings")
         for required_field in schema.get("required_fields", []):
             field_name = required_field.get("name")
-            field_value = incoming_settings.get(field_name)
+            field_value = next(
```

Comment:
Although this is a clean one-liner. It would make more sense to extract this logic into a function that can be reused below.